.github/process_commit.py

```diff
@@ -33,46 +38,47 @@
 }
 
 
-def main(commit_hash: str) -> Dict[str, Any]:
+def find_responsible_users(commit_hash: str) -> Set[str]:
     pr_number = get_pr_number(commit_hash)
+    if not pr_number:
+        return set()
+
     merger, labels = get_pr_merger_and_labels(pr_number)
     is_properly_labeled = bool(REQUIRED_LABELS.intersection(labels))
-    if not is_properly_labeled:
-        users = {merger, *get_pr_reviewers(pr_number)}
-    else:
-        users = ()
-    return dict(
-        is_properly_labeled=is_properly_labeled,
-        responsible_users=", ".join(sorted([f"@{user}" for user in users])),
-    )
+    if is_properly_labeled:
+        return set()
+
+    return {merger, *get_pr_reviewers(pr_number)}
```

Comment:
nit: I'd just ping the merger, but it's not important

Reply:
Sometimes whoever merged it, did not dive deep into the PR, but rather relied on previous reviews. Note that not all reviewers, but only reviewers that approved the PR will get pinged:

https://github.com/pytorch/vision/blob/4db4d157d919d869888460f8473f33dfd6a30bfc/.github/process_pr.py#L72